FreeRTOS/FreeRTOS-Plus-TCP
# Add Ethernet driver for Corstone-300 FVP (MPS3_AN552)

Add Ethernet driver for Corstone-300 FVP (MPS3_AN552)

Description
-----------

The Corstone-300 FVP models SMSC 91C111 Ethernet controller. Add a network interface based on CMSIS ethernet driver for SMSC 91C111.

Test Steps
-----------
Tested locally with Arm Featured Reference Integration https://github.com/FreeRTOS/iot-reference-arm-corstone3xx. Once this PR is merged, FreeRTOS TCP/IP stack support for https://github.com/FreeRTOS/iot-reference-arm-corstone3xx will be created.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
